src/main/java/com/example/semanews/ArticleEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.example.semanews;
+
+import org.springframework.data.neo4j.core.schema.Id;
+import org.springframework.data.neo4j.core.schema.Node;
+
+@Node("Article")
+public class ArticleEntity {
+    @Id
+    private final String name;
+    private final String url;
+
+    public ArticleEntity(String name, String url) {
+        this.name = name;
+        this.url = url;
```

Comment:
https://docs.oracle.com/javase/jp/8/docs/api/java/net/URL.html
you should use `java.net.URL`.